pontoon/base/views.py

```diff
@@ -261,12 +261,13 @@ def translate(request, locale, slug, part=None, template='translate.html'):
         except Subpage.DoesNotExist:
             # If page not specified or doesn't exist
             page = pages[0]
-            locale_pages = pages.filter(resources__stats__locale=l)
```

Comment:
Yeah, ignore me, there's no better way than what you've done without a more serious refactoring.